wagtail_bucketav/utils.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from wagtail.documents import get_document_model
+from wagtail.images import get_image_model
+
+from .models import FileScanStatus
+
+logger = logging.getLogger("wagtail_bucketav")
+
+
+def get_object_for_key(file_key):
+    image_for_file = get_image_model().objects.filter(file=file_key).first()
+
+    if image_for_file:
+        return image_for_file
+
+    return get_document_model().objects.filter(file=file_key).first()
```

Comment:
This would be cool, but the _Wagtail_ part of this is for images and documents. Adding more generic model integration makes it really more of a Django integration than a Wagtail one (I'd kinda been thinking about extracting or moving the wagtail stuff to a `contrib` package).

Reply:
I understand, but wagtail sites very often have other models that you'd want to be scannable too, so baking that in here would be sensible I think. For example, `wagtail_media`'s `Media` model would be required in most of the larger Torchbox. sites.

Reply:
Happy for this to be implement later if you just want to get the basic version in and working?

Reply:
Given it's pretty trivial, I'll give it a shot!